CompanyNormalization.md

```diff
@@ -50,10 +52,10 @@ Example: https://api.careerbuilder.com/core/normalizedcompanies?company_name=car
 
 #Response Information
 
-The response returns a single data node which contains a list of normalized companies. These normalized companies are ordered by the confidence score. Each normalized company has a normalized company name (string), a naics ID (string), a naics description (string), a country (string), a state(string), a city(string), a postal code(string), and a confidence (double). Confidence scores range from 0.0 to 1.0.
+The response returns a single data node which contains a list of normalized companies. These normalized companies are ordered by the confidence score. Each normalized company has a normalized company name (string), a naics ID (string), a naics description (string), a duns number(string), a country (string), a state(string), a city(string), a postal code(string), a website(string), and a confidence (double). Confidence scores range from 0.0 to 1.0.
```

Comment:
let's capitalize DUNS number to be clear that it's [an acronym](https://en.wikipedia.org/wiki/Data_Universal_Numbering_System)